Website/wwwroot/css/bootstrap/bootstrap.min.css

```
@@ -1,4 +1,4 @@
-@charset "UTF-8";/*!
+Get@charset "UTF-8";/*!
```

```suggestion
@charset "UTF-8";/*!
```
this looks like a typo